core/roboconf-dm/src/main/java/net/roboconf/dm/internal/delegates/InstanceMngrDelegate.java

```diff
@@ -400,6 +400,11 @@ private void deployTarget( ManagedApplication ma, Instance scopedInstance ) thro
 				throw (TargetException) e;
 			else if( e instanceof IOException )
 				throw (IOException) e;
+
+		} finally {
+			synchronized( LOCK ) {
+				scopedInstance.data.remove( Instance.TARGET_ACQUIRED );
```

Comment:
I think this line make the CI builds fail. Maybe the `Manager_LifeCycleTest.testDeployRootMoreThanOnce()` test need a little update too?
```
Results :

Failed tests: 
  Manager_LifeCycleTest.testDeployRootMoreThanOnce:339 null

Tests run: 136, Failures: 1, Errors: 0, Skipped: 0
```